docker-compose.yml

```diff
@@ -53,3 +53,13 @@ services:
       - |
         pip install --no-cache-dir --upgrade -r requirements.txt
         uvicorn app.server:app --host $$HOST --port $$PORT --workers $$WORKERS
+
+  frontend:
+    image: httpd:2.4
+    volumes:
+      - ./Frontend:/usr/local/apache2/htdocs/
+      - ./my-httpd.conf:/usr/local/apache2/conf/httpd.conf
+    ports:
+      - "80:80"

```

Comment:
you shouldn't use port 80 cause it will clash with the current nginx config

Reply:
What ports are available for us?